src/pages/Account/RequestResetPassword/RequestResetPassword.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { ChangeEvent, FormEvent, useState } from 'react';
+
+import { Link } from 'react-router-dom';
+
+import requestResetPassword from '@/apis/requestResetPassword';
+import { MESSAGES, STATUS_CODES } from '@/constants/constantValues';
+import { emailValidationPattern } from '@/helpers/validators';
+
+import type { IAccountInfo } from '../types';
+
+export default function RequestResetPassword(): JSX.Element {
+  const [email, setEmail] = useState<string>('');
+  const [emailValidationMessage, setEmailValidationMessage] = useState<string>('');
+  const [resetPasswordInfo, setResetPasswordInfo] = useState<IAccountInfo>({
+    code: STATUS_CODES.OK,
+    message: '',
+  });
+
+  const handleEmail = (event: ChangeEvent<HTMLInputElement>): void => {
+    setEmailValidationMessage('');
+    setEmail(event.target.value);
+  };
+
+  const validateEmail = (email: string): boolean => {
```

Comment:
Should this custom validation be handled using yup? Or it's intentionally implemented?

Reply:
Yes, it is intentionally implemented as we said that one form will be created without using any additional packages.